### ISSUE TITLE
Crashes on iOS

### ISSUE THREAD

**Issue by ghost** [fuse-compound/Fuse.StreamingPlayer#3]
Andriod works fine but carshes on iOS.

And xcode gives error on :
NowPlayingInfoCenter.uno:62

[Foreign(Language.ObjC)]
static ObjC.Object CreateArtworkFromUrl(string url)
@{
if (url == null) return null;

**Comment by ikalangita** [comment]
You have just comment this line : if (url == null) return null; when xcode
shows errors, and it works

2018-02-16 13:46 GMT+01:00 Victor Wallander <notifications@github.com>:

**Comment by ghost** [comment]
@ikalangita  or you change from null to NULL as it should be

**Comment by ikalangita** [comment]
Yes, and the best way is to change it from fuse not inside xcode

2018-02-17 10:06 GMT+01:00 Victor Wallander <notifications@github.com>:

**Comment by ghost** [comment]
But It seems not to be working to change it to NULL in Fuse?

**Comment by ikalangita** [comment]
Don't know, i just commented this line

2018-02-17 12:27 GMT+01:00 Victor Wallander <notifications@github.com>:
